src/entities/Proposal/templates/index.ts

```diff
@@ -18,7 +19,11 @@ type NewConfiguration =
 export const title = async ({ type, configuration }: { type: ProposalType, configuration: NewConfiguration }) => {
   switch (type) {
     case ProposalType.POI:
-      return poi.title(configuration as any)
+      return "" // no title
```

Comment:
all this new cases wouldn't be needed because the `title`/`description`/`etc`/ fn inside addPoi can determine what to do based on the `NewProposalPoi` object